tutorial/sample-problems/Algebra/AnswerBlankInExponent.pg

```diff
@@ -1,3 +1,4 @@
+## Please see file perltidy.ERR
```

Comment:
Is this line supposed to be here?

Reply:
Nope.

Reply:
I force pushed the removal of that line.

Reply:
Thanks for finding that.